examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -847,7 +847,9 @@ class WebGPURenderer {
 
 		const drawRange = geometry.drawRange;
 		const firstVertex = drawRange.start;
-		const instanceCount = ( geometry.isInstancedBufferGeometry ) ? geometry.instanceCount : 1;
+
+		let instanceCount = object.isInstancedMesh ? object.count : 1;
+		instanceCount = geometry.isInstancedBufferGeometry ? geometry.instanceCount : instanceCount;
```

Comment:
I think it would be cleaner to write `instanceCount = geometry.isInstancedBufferGeometry ? geometry.instanceCount : ( object.isInstancedMesh ? object.count : 1 )`.